### ISSUE TITLE
Interest in supporting custom endpoints

### ISSUE THREAD

**Issue by rmanyari** [googleapis/nodejs-compute#53]
Now that big codebase split seems to be completed I was wondering if there's interest in supporting custom endpoints as discussed in [1630](https://github.com/GoogleCloudPlatform/google-cloud-node/issues/1630) and implemented in [2548](https://github.com/GoogleCloudPlatform/google-cloud-node/pull/2548)

I'm particularly interested in the compute module since it's the one we use heavily. If it's the case, I can get a PR ready in the next few days.

Rodrigo

**Comment by stephenplusplus** [comment]
Yeah, let's go for it! I think we need to get a clear goal in https://github.com/GoogleCloudPlatform/google-cloud-node/issues/1630 before we start, so let's put this on hold while we discuss.

**Comment by JustinBeckwith** [comment]
Good news!  This was added in https://github.com/googleapis/nodejs-compute/commit/49c53847b3e5dd993a3996a80ad0fce798ba092c.
